README.md

```diff
@@ -271,3 +274,35 @@ ENTRYPOINT ["/go-binary"]
 ### Build final docker image
 
 `docker build -t docker-edgex-binary -f Dockerfile --build-arg BASE=base-build .`
+
+## Deployment On Sandbox
+
+
+The official site for Sandbox environment is **[https://jenkins.edgexfoundry.org/sandbox/](https://jenkins.edgexfoundry.org/sandbox/)**
+
+#### Get Access To Sandbox
+
+To deploy the pipeline on test environment, You need to request for access to sanbox through **[LF Service Desk](https://jira.linuxfoundation.org/servicedesk/customer/portals)**
+
+If you already have the LFID and can access the Prod Environment you can use the same LFID to access the Sandbox.
+
+#### Implementation
+
+Once you create a new jenkins file follow the procedure as above. In your github repository, your jenkinsfile name should be Jenkinsfile.sanbox. Since, the jenkins sanbox will recognizes only the file Jenkinsfile.sanbox to execute.
```

Comment:
spelling eror - please change to correct spelling - "sandbox"

Reply:
I think we wanted to get away from using a separate Jenkinsfile based on some initial testing.

Reply:
what i understood is we using the same jenkinsfile for box prod and sandbox but we just changing the name of the file right!